fourth-challenge/public/js/cities/form.js

```diff
@@ -0,0 +1,66 @@
+$(document).ready(function() {
+    console.log("Creando una ciudad");
+    $('.create-form').on('submit', function(e) {
+        e.preventDefault();
+
+        const formData = $(this).serialize();
+        $.ajax({
+            type: 'POST',
+            url: 'cities',
+            data: formData,
+            success: function(data) {
+                $('#table-container').html(data);
+                const newPaginationHtml = $(data).find('#pagination-container').html();
+                $('#pagination-container').html(newPaginationHtml);
+
+                // Reset the form inputs
+                $('#new-city-name').val('');
+                // $('#flights-arriving').val('');
+                // $('#flights-departing').val('');
+            },
+            error: function(xhr, status, error) {
```

Comment:
As a general rule, I would not create any pr with todos if I can help it. Either resolve all the todos before the request or those you won't resolve now can be loaded to the management system of choice